common/message_types.txt

```diff
@@ -1,1711 +0,0 @@
-message_type = {
```

Comment:
I want my message type

Reply:
![image](https://user-images.githubusercontent.com/17663470/79696205-46301080-8249-11ea-8769-d5c8053c16c5.png)

You have your message type.

Reply:
When you abduct a Pop using the Pirate Raid, the custom message fires. I can show you a screen shot of it in the game.

Reply:
i want my message back

Reply:
Do I REALLY have to show you an actual screenshot of the message firing in the game? Why don't you believe me? I have abducted Pops in the game using the Pirate Raid system. I know for a fact that it works.

Reply:
I dont want vanilla message, i want my own